invoice2data/input/tesseract.py

```diff
@@ -14,14 +14,21 @@ def to_text(path):
 
     """
     import subprocess
+    from distutil import spawn
```

Comment:
Here is a typo. The package is called `distutils`. Note the extra "s" at the end.